lib/spack/spack/cmd/style.py

```diff
@@ -113,12 +135,6 @@ def setup_parser(subparser):
         action="store_true",
         help="check all files, not just changed files",
     )
-    subparser.add_argument(
-        "-o",
-        "--output",
-        action="store_true",
-        help="send filtered files to stdout as well as temp files",
-    )
```

Comment:
Should this be deprecated before removing?

Reply:
tempted to say no as I do not think anyone uses it for anything except for (maybe?) us.  Also it was already not working or doing anything.